core/pkg/resources/careq.go

```diff
@@ -62,7 +62,7 @@ type ImportCABody struct {
 }
 
 type UpdateCAMetadataBody struct {
-	Metadata map[string]interface{} `json:"metadata"`
```

Comment:
@haritzsaiz This is a general comment, how can we document the major API changes? We need to have a place to document these breaking changes.